src/main/java/mate/jdbc/model/Driver.java

```diff
@@ -0,0 +1,55 @@
+package mate.jdbc.model;
+
+public class Driver {
+    private Long id;
+    private String name;
+    private String licenseNumber;
+
+    public Driver() {
+    }
+
+    public Driver(String name, String licenseNumber) {
+        this.name = name;
+        this.licenseNumber = licenseNumber;
+    }
+
+    public Driver(Long id, String name, String licenseNumber) {
+        this.id = id;
+        this.name = name;
+        this.licenseNumber = licenseNumber;
+    }
```

Comment:
Reuse contractor with 2 parameters
```suggestion
    public Driver(Long id, String name, String licenseNumber) {
        this(name, licenseNumber);
        this.id = id;
    }
```